src/daidepp/constants.py

```diff
@@ -0,0 +1,170 @@
+from typing import Literal
```

Comment:
This doesn't work in Python 3.7. You need to use `from typing_extensions import Literal`